Packages/vcs/vcs/VCS_validation_functions.py

```diff
@@ -1745,13 +1745,37 @@ def _setpriority(self, value):
 priority = property(_getpriority, _setpriority)
 
 
-def _getX1(self):
+@property
+def arrow(self):
+    """length of extension arrows"""
+    return self._arrow
+
+
+@arrow.setter
+def arrow(self, value):
+    self._arrow = checkNumber(self, "arrow", value, 0., None)
+
+
+@property
+def offset(self):
+    """offset position in % of page (can be negative)"""
+    return self._offset
```

Comment:
@doutriaux1 Can you expand the comment for offset to include the detailed explanation in the comments section?